cursor/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "cursor",
```

Comment:
CodeJar is small for a resulting bundle. There is no benefit in package size. 

Why this configuration file? It adds too much bookkeeping without any benefits. 

I like what CodeJar currently requires as little as possible (only typescript).

Reply:
This config file is needed for microbundle to bundle "sub" packages, like preact do.
So an user can import cursor by `import {cursor} from "@medv/codejar/cursor"`

Btw, i know that this solution is over engineering, microbundle doesn't support different input files yet.